src/ruby/spec/basic_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe SymEngine do
+    before :each do
+    end
+
+    describe SymEngine::Basic do
+        before :each do
+        end
+
+        describe '.new' do
+            context 'with no arguments' do
+                it 'returns a Basic object' do
+                    expect(SymEngine::Basic.new).to be_an_instance_of SymEngine::Basic
+                end
+            end
+        end
+    end
+end
```

Comment:
I think you should also test operations you defined src/ruby/ext/symengine/symengine.c .